### PR TITLE
Feat/login and signup

### PR DESCRIPTION
**Pull request by tonymike99**
Hey! I've added the login and signup pages.

https://serene-swartz-16d893.netlify.app/

image

image

### CODE REVIEW

File: templates/signup.html [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+            <header class="header">
+                <div class="header-item">
+                    <a class="brand-name" href="/index.html">Candelify</a>
+                </div>
+
+                <nav class="header-item">
+                    <ul class="list list-spaced list-navbar">
+                        <li>
+                            <a class="styled-link" href="/templates/login.html"
+                                >Login/Signup</a
```
Comment (humzahasan) [inline templates/signup.html:35]: On hover maybe add a bit more bright color, the last color is blending with UI

**Review by humzahasan — APPROVED** [review]
Feedback :
- Good use of comments for documentation
- Good selection of descriptive class name


UI Suggestion : On hover the text blending with dark background color, you can look into that and refactor.

Good to merge!

File: templates/signup.html (hunk 1 above)
Comment (tonymike99) [reply templates/signup.html:35]: > On hover maybe add a bit more bright color, the last color is blending with UI

Sure. Will do. Thanks Humza.

**Review by developer-anand — APPROVED** [review]
Good to merge.

**Pull request merged**